api/python/notebooks/api_demo/census_duplicated_cells.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Understanding and filtering out duplicate cells\n",
+    "\n",
+    "This tutorial provides an explanation for existence of duplicate cells in the Census, and it showcases different ways to handle these cells when performing queries on the Census using the `is_primary_data` cell metadata variable. \n",
+    "\n",
+    "**Contents**\n",
+    "\n",
+    "1. Why are there duplicate cells in the Census?\n",
+    "2. An example: duplicate cells in the Tabula Muris Senis data.\n",
+    "3. Filtering out duplicates cells.\n",
+    "   1. Filtering out duplicate cells when reading the `obs` data frame.\n",
+    "   2. Filtering out duplicate cells when creating an AnnData.\n",
+    "   3. Filtering out duplicate cells for out-of-core operations.\n",
+    "   \n",
+    "## Why are there duplicate cells in the Census?\n",
+    "\n",
+    "Duplicate cells are labeled on the `is_primary_data` cell metadata variable as `False`. To learn more about this please take a look at the corresponding [section of the dataset schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#is_primary_data). \n",
+    "\n",
+    "The Census data is a concatenation of most RNA data from CZ CELLxGENE Discover and these data are ingested one dataset at a time. You can take a look at what data is included in the Census [here](https://chanzuckerberg.github.io/cellxgene-census/cellxgene_census_docsite_schema.html).\n",
+    "\n",
+    "In some cases data from the same cell exists in different datasets, therefore cells can be duplicated throughout CELLxGENE Discover and by extension the Census. \n",
+    "\n",
+    "The following are a few examples where cells are duplicated in CELLxGENE Discover:\n",
+    "\n",
+    "* There are superset datasets containing data from multiple datasets.\n",
+    "> *For example [Tabula Sapiens](https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5) has one dataset with all of its cells and separate datasets with cells divided by high-level lineage (i.e. immune, epithelial, stromal, endothelial)*\n",
+    "* There are datasets with meta-analysis of pre-existing datasets.\n",
+    "> *For example [Jin et al.](https://cellxgene.cziscience.com/collections/b9fc3d70-5a72-4479-a046-c2cc1ab19efc) performed a meta-analysis of COVID-19 data, and they included both the individual datasets as well as one concatenated dataset*\n",
+    "\n",
+    "The Census has all of these data to allow for the execution of dataset-based queries, which would be otherwise limited if only non-duplicate cells were included.\n",
```

Comment:
```suggestion
    "The Census has all of these data to allow for the execution of dataset-based queries, which would be otherwise be limited if only non-duplicate cells were included.\n",
```